From: Eugene Gallagher <eugene_gallagher@comcast.com>
Date: 04/23/2021
Subject: [PATCH] add debug logging to bluez daemon

add -d option when starting the daemon to enable debug logs

Upstream-Status: N/A
Source: Comcast
License: CLOSED
Signed-off-by: Eugene Gallagher <eugene_gallagher@comcast.com>

diff -Naur a/src/bluetooth.service.in b/src/bluetooth.service.in
--- a/src/bluetooth.service.in	2021-04-23 16:59:08.012263158 +0000
+++ b/src/bluetooth.service.in	2021-04-23 14:37:16.082406468 +0000
@@ -10,7 +10,7 @@
 BusName=org.bluez
 ExecStartPre=/bin/mkdir -p /var/lib/bluetooth
 ExecStartPre=/bin/mkdir -p /opt/lib/bluetooth
-ExecStart=@libexecdir@/bluetoothd
+ExecStart=@libexecdir@/bluetoothd -d
 NotifyAccess=main
 #WatchdogSec=10
 Restart=always
